src/feat/pitch-functions.h

```diff
@@ -198,12 +198,16 @@ struct PitchExtractionOptions {
   }
   /// Returns the window-size in samples, after resampling.  This is the
   /// "basic window size", not the full window size after extending by max-lag.
+  // B.P. fixed a bug here. Even though the result of the operation is supposed 
+  // to be an integer, you still need rounding before casting :-)
+  // NB: 0.001 is impossible to represent exactly on a float or double, so for
+  // this to work without rounding, you would need to divide by 1000 instead.
   int32 NccfWindowSize() const {
-    return static_cast<int32>(resample_freq * 0.001 * frame_length_ms);
+      return static_cast<int32>(round(resample_freq * 0.001f * frame_length_ms));
```

Comment:
you have added unnecessary extra indentation here.
Can you explain why you feel this was a bug?  By default, casting rounds down.  Why is rounding to closest more appropriate in your opinion?  Unless it was causing some kind of crash, best to leave it as it was.